pl_examples/quantization/intel_neural_compressor/README.md

```diff
@@ -0,0 +1,118 @@
+# Step-by-Step
+
+This document describes the step-by-step instructions for reproducing PyTorch ResNet50/ResNet18/ResNet101 tuning results with [Intel® Neural Compressor](https://github.com/intel/neural-compressor).
+
+# Prerequisite
+
+### 1. Installation
+
+```shell
+pip install neural-compressor
+pip install torch>=1.9.0 torchvision>=0.10.0 --find-links https://download.pytorch.org/whl/torch_stable.html
+```
+
+### 2. Prepare Dataset
+
+Download [ImageNet](http://www.image-net.org/) raw image to the following directory: /path/to/imagenet.  The directory should include a train and val folders:
+
+```bash
+ls /path/to/imagenet
+train  val
+```
+
+# Quantize the model
+
+### Write Yaml Config File
+
+```yaml
+model:
+  name: imagenet
+  framework: pytorch_fx                              # supported values are pytorch_fx, pytorch_ipex here.
+
+device: cpu                                          # Now only support CPU device, will support Intel GPU future.
+
+quantization:
+  approach: post_training_static_quant               # supported values are post_training_static_quant, post_training_dynamic_quant, quant_aware_training.
+  calibration:
+    sampling_size: 300
+
+tuning:
+  accuracy_criterion:
+    relative:  0.01                                  # optional. default value is relative, other value is absolute. this example allows relative accuracy loss: 1%.
+  exit_policy:
+    timeout: 0                                       # optional. tuning timeout (seconds). default value is 0 which means early stop. combine with max_trials field to decide when to exit.
+    max_trials: 1200
+  random_seed: 9527                                  # optional. random seed for deterministic tuning.
+```
+
+- For "framework" filed, the "pytorch_fx" is for FX mode, and "pytorch_ipex" is for intel extension of PyTorch.
+
+- For quantization approach, Please refer below link:
+  [PyTorch Quantization](https://pytorch.org/docs/stable/quantization.html)
+
+### Saving and loading model:
+
+- Saving quantization configure and weights:
+  After tuning with Neural Compressor, we can get neural_compressor.model:
+
+```
+cli = MyLightningCLI(ImageNetLightningModel,
+                     seed_everything_default=42,
+                     save_config_overwrite=True, run=False)
+callback = pl.callbacks.INCQuantization(
```

Comment:
This can be added in the trainer_defaults.

Reply:
done